code/modules/uplink/uplink_items.dm

```diff
@@ -1892,10 +1892,10 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	cost = 14
 	restricted_roles = list("Geneticist", "Research Director")
 
-/datum/uplink_item/role_restricted/chemical_gun
-	name = "Reagent Dartgun"
-	desc = "A heavily modified syringe gun which is capable of synthesizing its own chemical darts using input reagents. Can hold 100u of reagents."
-	item = /obj/item/gun/chem
+/datum/uplink_item/role_restricted/rapid_syringe_gun
+	name = "Rapid Syringe Dartgun"
+	desc = "A heavily modified syringe gun which is capable of holding up to 6 syringes for rapid chemical payload delivery direct to the target."
+	item = /obj/item/gun/syringe
 	cost = 12
```

Comment:
drop this shit to like, 5 or something, nobody is spending over half their TCs on a syringe gun that you can't hide in any way

Reply:
Like many other traitor items, it fits in your backpack and if you don't want to be caught with it, don't carry it on yourself 24/7.

It's a big TC investment, but also nothing else in the game can rapidly deliver 6 bluespace syringes of chems into 1-6 targets within the space of a few seconds.

Reply:
this is an abysmal item to purchase for 12 fuckin tc, lower it and stop pretending people are gonna buy it at 12 tc

Reply:
for 12 tc i could get an esword which is significantly more deadly

i can then use that esword to get the hypospray

and then i can deliver way more chems into someone, then follow it up with the esword

and i can put that esword into my fuckin pocket

Reply:
Rapid syringe gun is really good I think this price is pretty fair considering landing one syringe on someone will kill them considering you’re competent